### ISSUE TITLE
add a sub predicate of correlated with (literature co-occurrence?) that would capture the idea that two things co-occur in a threshold of publications

### ISSUE THREAD

**Issue by sierra-moxon** [biolink/biolink-model#882]
ARA: Aragorn
Workflow: A
PGR->TrichtostatinA (correlated with currently used)
has_count: omnicorp_article_count
value: 11

would like to be able to use the new predicate here instead.

@cbizon - could you confirm that I've captured this use case from the relay correctly?  :)

**Comment by cbizon** [comment]
Yes, that is correct.  See also https://github.com/biolink/biolink-model/issues/58.

The other thing worth noting (perhaps) is that we also add a related attribute to nodes, which is the total number of articles that concept appears in.  That allows us to normalize the value in this edge.
